app/scripts/views/elements/README.md

```diff
@@ -0,0 +1,14 @@
+# Custom Elements
+
+Contents of this directory are to define custom element helpers
+for view elements. Each module should define at minimum two functions:
+
+`match` - check whether the current module should be used with the given element.
+`validate` - validate the element. Return a falsy value if valid, and an Error
+if invalid.
```

Comment:
Before reading the description, I was expecting this to return `true` if the element is valid, `false` otherwise. Or maybe it would throw in the error case. Returning falsey in the success case feels a bit surprising to me, given the method name.

It's a bit of a mouthful but something along the lines of `getValidationErrors` strikes me as being more precise for the current semantics.